### PR TITLE
Contact 'page'

### PR DESCRIPTION
**Pull request by Amondt**
(it's actually a modal available everywhere on the website, via the navbar)

### CODE REVIEW

**Comment by Amondt** [comment]
I also added in the main 'reset.css' some font improvements (anti-aliasing)

**Comment by Amondt** [comment]
The automation emails still has to be done, I got a call later today with activeCampaign for it

File: store/settings.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const state = () => ({
   primaryNavigation: [],
   cacheVersion: '',
-  editMode: false,
-  version: 'published',
+  editMode: true,
```
Comment (shven) [inline store/settings.js]: please revert

File: layouts/default.vue [hunk 2]
```diff
@@ -403,6 +426,26 @@
...
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.9);
+    overflow-y: auto;
+    // opacity:0;
```
Comment (shven) [inline layouts/default.vue]: cleanup comments

Comment (Amondt) [reply layouts/default.vue]: @shven done

**Pull request merged**